### ISSUE TITLE
No backlink to Dissemin from association.dissem.in

### ISSUE THREAD

**Issue by Phyks** [dissemin/dissemin#680]
When clicking "Who we are", the user is sent to http://association.dissem.in/. From there, there is no backlink to Dissemin (actually, there is one hidden in the small heading text).

Maybe after the first two lines, there could be a large clickable Dissemin logo?

Maybe this page should be somehow embedded into dissemin like the "Partners"?

**Comment by wetneb** [comment]
I agree with the backlink, I'll fix that.

I don't think this page should be in Dissemin's code base: the association and the software are distinct things, and the goal is that people can run the software without having any connection to the association (typically: OpenIng). Potentially the association could also have activities which are not strictly speaking about the software too.

**Comment by Phyks** [comment]
> I don't think this page should be in Dissemin's code base: the association and the software are distinct things

Indeed, that makes sense. :) I was thinking about an "authors" page, but that could be an issue for other instances.

**Comment by wetneb** [comment]
Actually, the backlink seems fairly visible to me at the moment, and we already have one logo for the association, so I am not sure what to change. I'll give you the credentials to edit if you have a precise idea of how it should look like.
